### ISSUE TITLE
(MacOS) Option-Enter Replace sometimes places the UUID incorrectly

### ISSUE THREAD

**Issue by babramovitch** [leomillon/uuid-generator-plugin#133]
When I try to replace the gen command with a new UUID using Option-Enter to speed things up, it sometimes places the new UUID incorrectly.  This doesn't happen if I mouse click on the content, then choose additional actions and then choose replace.

This is on version 4.5.1

![Screen Shot 2022-05-05 at 12 12 04 PM](https://user-images.githubusercontent.com/3608443/167008114-9d96635b-571b-43d9-9170-967c4e70b2cf.png)

into

![Screen Shot 2022-05-05 at 12 10 31 PM](https://user-images.githubusercontent.com/3608443/167007699-71d317b5-246e-43e3-a39c-046e6eff11b1.png)

It's pretty frequent, like 1 in 4 are wrong

**Comment by leomillon** [comment]
Hi @babramovitch!

Thanks for the feedback, but I'm not able to reproduce this error... 😞

Could you please share at least:
- the version of IntelliJ IDEA you're using
- a sample projet / use-case to reproduce this exact behavior

**Comment by MartinX3** [comment]
Same problem on linux.
It seems to happen in high cpu usage situations with the IDE.

**Comment by babramovitch** [comment]
@leomillon I missed that you'd replied to this, but just saw MartinX3's notification.  

It was happening before and up to on this.  I'll see if I can reproduce it sometime in the next couple days in a sample project for you, as well as on any newer versions if there is.

Android Studio Chipmunk | 2021.2.1 Patch 1
Build #AI-212.5712.43.2112.8609683, built on May 18, 2022
Runtime version: 11.0.12+0-b1504.28-7817840 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 12.2
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 10
Registry: external.system.auto.import.disabled=true
Non-Bundled Plugins: wu.seal.tool.jsontokotlin (3.7.4), com.github.leomillon.uuidgenerator (4.5.1), org.jetbrains.kotlin (212-1.7.0-release-281-AS5457.46), izhangzhihao.rainbow.brackets (6.25)

**Comment by MartinX3** [comment]
Probably the project is dead and the author lost in the void due to lack of free time.

**Comment by leomillon** [comment]
@MartinX3 you're right, I don't have much time to spend on this project right now BUT that does not mean that this projet is dead.

If you want to help solving this, please provide "a sample projet / use-case to reproduce this exact behavior".

And don't forget that you can also open a pull-request to fix it ;)

**Comment by MartinX3** [comment]
Thank you for your info. :)

Sadly it's hard to reproduce.
I need to have a large file and a high cpu usage in IntelliJ.
Then I need to fast replace many uuid by hand with the `Replace at cursor` action, maybe with a macro.
And then it happens rarely here.
Or maybe he can trigger it because the IntelliJ is slow due to his low RAM.

**Comment by leomillon** [comment]
The only thing I can think of is some "miss alignment" with characters indices when virtual hints are displayed in the document (changing virtually the characters indices).

That's why it would be nice to find a use-case easy to reproduce :/
